src/reducers/index.js

```diff
@@ -124,6 +125,7 @@ const rootReducer = combineReducers({
     userLocation: userLocationReducer,
     userLocationAccessGranted: userLocationAccessGrantedReducer,
     showReportHeatmap: reportHeatmapStateReducer,
+    schemaSelector: schemaSelectorReducer,
```

Comment:
A couple TODOs here and in the duck file would be good to remember to delete the reducer after we are done.